gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsInstrumentation.java

```diff
@@ -200,6 +208,22 @@ public void incrementCounter(GhfsStatistic op, long count) {
     instanceIOStatistics.incrementCounter(name, count);
   }
 
+  /**
+   * Increments a mutable counter and the matching instance IOStatistics counter. No-op if the
+   * counter is not defined, or the count == 0.
+   *
+   * @param op operation
+   * @param count increment value
+   */
+  public void incrementCounter(GcsStatusMetrics op, long count) {
```

Comment:
Since the count is always one, should we remove that? We can add them later if required.